unlock-app/src/middlewares/currencyConversionMiddleware.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
```

Comment:
I think here would be a good place to indicate that we could use `Fetch` if there was a larger support so that we can reconsider this in the future when support for Fetch is large enough.

Another option would also to be using a Polyfill since we already use [Babel](https://babeljs.io/docs/en/babel-polyfill) which could replace `Fetch` when it does not exist?

Reply:
added a TODO comment here

Reply:
Thanks!